samples/web/content/apprtc/js/util.js

```diff
@@ -119,3 +124,46 @@ function randomString(strLength) {
   }
   return result.join('');
 }
+
+// Returns true if the code is running in a packaged Chrome App.
+function isChromeApp() {
+  return (typeof chrome !== 'undefined' &&
+          typeof chrome.storage !== 'undefined' &&
+          typeof chrome.storage.local !== 'undefined');
+}
+
+// Get a value from local browser storage. Calls callback with value.
+function getStorage(key, callback) {
+  if (isChromeApp()) {
+    // use chrome.storage.local
+    chrome.storage.local.get(key, function(values) {
+      // unwrap key/value pair
+      if (callback)
+      {
+        callback(values[key]);
+      }
+    });
+  } else {
+    // use localStorage
+    var value = localStorage.getItem(key);
+    if (callback) {
+      callback(value);
```

Comment:
Calling callbacks from within the actual function can lead to reentry. Consider calling this from setInterval.